trace_log.log

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Let's gitignore this file (and also rename it to trace.log)

Reply:
added git ignore to all .log files